templates/takeovers/_ubuntu-masters-virtual-202004.html

```diff
@@ -0,0 +1,15 @@
+{% with title="Join the virtual Ubuntu Masters event",
```

Comment:
This title doesn't match the title in the copy doc?
"Join the Ubuntu Masters virtual event"

Reply:
I wrote both... what sounds better?